lib/features/reps_core/reps_core.module

```diff
@@ -24,6 +24,17 @@ function reps_core_help($path, $arg) {
   return $help;
 }
 
+/**
```

Comment:
I'm not sure this code indent will pass drone automate tests ... 
But it's more readable.